src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -632,6 +648,7 @@ ProcessCLRException(IN     PEXCEPTION_RECORD   pExceptionRecord,
     }
 
 #ifndef HOST_UNIX
+    // First pass (filtering)
```

Comment:
A nit:
I would change "filtering" to "searching" and in the second pass comment "handling" to "unwinding".